src/flamenco/features/feature_map.json

```diff
@@ -162,34 +161,34 @@
   {"name":"reduce_stake_warmup_cooldown","pubkey": "GwtDQBghCTBgmX2cpEGNPxTEBUTQRaDMGTr5qychdGMj","hardcoded":1},
   {"name":"revise_turbine_epoch_stakes","pubkey": "BTWmtJC8U5ZLMbBUUA1k6As62sYjPEjAiNAT55xYGdJU"},
   {"name":"enable_poseidon_syscall","pubkey": "FL9RsQA6TVUoh5xJQ9d936RHSebA1NLQqe3Zv9sXZRpr"},
-  {"name":"timely_vote_credits","pubkey": "2oXpeh141pPZCTCFHBsvCwG2BtaHZZAtrVhwaxSy6brS"},
+  {"name":"timely_vote_credits","pubkey": "tvcF6b1TRz353zKuhBjinZkKzjmihXmBAHJdjNYw1sQ"},
   {"name":"remaining_compute_units_syscall_enabled","pubkey": "5TuppMutoyzhUSfuYdhgzD47F92GL1g89KpCZQKqedxP"},
   {"name":"enable_program_runtime_v2_and_loader_v4","pubkey": "8oBxsYqnCvUTGzgEpxPcnVf7MLbWWPYddE33PftFeBBd"},
   {"name":"require_rent_exempt_split_destination","pubkey": "D2aip4BBr8NPWtU9vLrwrBvbuaQ8w1zV38zFLxx4pfBV"},
   {"name":"better_error_codes_for_tx_lamport_check","pubkey": "Ffswd3egL3tccB6Rv3XY6oqfdzn913vUcjCSnpvCKpfx"},
-  {"name":"enable_alt_bn128_compression_syscall","pubkey": "EJJewYSddEEtSZHiqugnvhQHiWyZKjkFDQASd7oKSagn"},
-  {"name":"programify_feature_gate_program","pubkey": "8GdovDzVwWU5edz2G697bbB7GZjrUc6aQZLWyNNAtHdg"},
   {"name":"update_hashes_per_tick2","pubkey":"EWme9uFqfy1ikK1jhJs8fM5hxWnK336QJpbscNtizkTU"},
   {"name":"update_hashes_per_tick3","pubkey":"8C8MCtsab5SsfammbzvYz65HHauuUYdbY2DZ4sznH6h5"},
   {"name":"update_hashes_per_tick4","pubkey":"8We4E7DPwF2WfAN8tRTtWQNhi98B99Qpuj7JoZ3Aikgg"},
   {"name":"update_hashes_per_tick5","pubkey":"BsKLKAn1WM4HVhPRDsjosmqSg2J8Tq5xP2s2daDS6Ni4"},
   {"name":"update_hashes_per_tick6","pubkey":"FKu1qYwLQSiehz644H6Si65U5ZQ2cp9GxsyFUfYcuADv"},
   {"name":"validate_fee_collector_account","pubkey":"prpFrMtgNmzaNzkPJg9o753fVvbHKqNrNTm76foJ2wm"},
+  {"name":"disable_rent_fees_collection","pubkey":"CJzY83ggJHqPGDq8VisV3U91jDJLuEaALZooBrXtnnLU"},
   {"name":"enable_zk_transfer_with_fee","pubkey":"zkNLP7EQALfC1TYeB3biDU7akDckj8iPkvh9y2Mt2K3"},
   {"name":"drop_legacy_shreds","pubkey":"GV49KKQdBNaiv2pgqhS2Dy3GWYJGXMTVYbYkdk91orRy"},
+  {"name":"allow_commission_decrease_at_any_time","pubkey":"decoMktMcnmiq6t3u7g5BfgcQu91nKZr6RvMYf9z1Jb"},
+  {"name":"add_new_reserved_account_keys","pubkey":"8U4skmMVnF6k2kMvrWbQuRUT3qQSiTYpSjqmhmgfthZu"},
   {"name":"consume_blockstore_duplicate_proofs","pubkey":"6YsBCejwK96GZCkJ6mkZ4b68oP63z2PLoQmWjC7ggTqZ"},
   {"name":"index_erasure_conflict_duplicate_proofs","pubkey":"dupPajaLy2SSn8ko42aZz4mHANDNrLe8Nw8VQgFecLa"},
-  {"name":"curve25519_restrict_msm_length","pubkey":"eca6zf6JJRjQsYYPkBHF3N32MTzur4n2WL4QiiacPCL"},
-  {"name":"allow_commission_decrease_at_any_time","pubkey":"decoMktMcnmiq6t3u7g5BfgcQu91nKZr6RvMYf9z1Jb"},
   {"name":"merkle_conflict_duplicate_proofs","pubkey":"mrkPjRg79B2oK2ZLgd7S3AfEJaX9B6gAF3H9aEykRUS"},
   {"name":"disable_bpf_loader_instructions","pubkey":"7WeS1vfPRgeeoXArLh7879YcB9mgE9ktjPDtajXeWfXn"},
-  {"name":"deprecate_executable_meta_update_in_bpf_loader","pubkey":"k6uR1J9VtKJnTukBV2Eo15BEy434MBg8bT6hHQgmU8v"},
```

Comment:
For features that have been cleaned up, could we set them to `"hardcoded":1`? Thanks 🙌

Reply:
fixed